server/service/src/sync/test/test_data/rnr_form_line.rs

```diff
@@ -34,10 +34,13 @@ fn rnr_form_line_1() -> RnRFormLineRow {
         item_id: "8F252B5884B74888AAB73A0D42C09E7A".to_string(),
         average_monthly_consumption: 0.0,
         initial_balance: 0.0,
-        quantity_received: 0.0,
-        quantity_consumed: 0.0,
+        snapshot_quantity_received: 0.0,
+        entered_quantity_received: None,
+        snapshot_quantity_consumed: 0.0,
+        entered_quantity_consumed: None,
         adjusted_quantity_consumed: 0.0,
-        adjustments: 0.0,
+        snapshot_adjustments: 0.0,
+        entered_adjustments: None,
```

Comment:
This test isn't fully fixed, just fixed the compilation part waiting for Mark's PR to get merged first.